tasks/add_new_prebid_partner.py

```diff
@@ -117,11 +127,17 @@ def setup_partner(user_email, advertiser_name, order_name, placements, ad_units,
   # Get the user.
   user_id = dfp.get_users.get_user_id_by_email(user_email)
 
-  # Get the placement IDs.
-  placement_ids = dfp.get_placements.get_placement_ids_by_name(placements)
-
-  # Get the ad unit IDs.
-  ad_unit_ids = dfp.get_ad_units.get_ad_unit_ids_by_name(ad_units)
+   # Get the placement IDs and ad_unit IDs.
+  placement_ids = None 
+  ad_unit_ids = None
+  if len(placements) > 0:
+      placement_ids = dfp.get_placements.get_placement_ids_by_name(placements)
+  elif len(ad_units) > 0:
```

Comment:
it should be `if` instead of `elif`- case when both adunits and placements are given by user